bad-code.py Test task for Python Mentor второе задание.py

```diff
@@ -0,0 +1,74 @@
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):

```

Comment:
you can add type annotation for `file_name` so it's would be understandable for the other developers what to expect. Also you can add annotation for the return value as well so everyone will understand what to expect in result